Resources/translations/SonataBlockBundle.en.xliff

```diff
@@ -2,24 +2,24 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
     <file source-language="en" datatype="plaintext" original="file.ext">
         <body>
-            <trans-unit id="sonata.block.container">
-                <source>sonata.block.container</source>
+            <trans-unit id="sonata.block.service.container">
```

Comment:
If we just remove the `.service` in `<service id="sonata.block.service...` we have a BC break.